solr/core/src/java/org/apache/solr/servlet/SolrRequestParsers.java

```diff
@@ -171,23 +169,22 @@ public SolrQueryRequest parse( SolrCore core, String path, HttpServletRequest re
     ArrayList<ContentStream> streams = new ArrayList<>(1);
     SolrParams params = parser.parseParamsAndFillStreams( req, streams );
 
-    Span span = (Span) req.getAttribute(Span.class.getName()); // not null but maybe in some tests?
-    if (span != null && !(span instanceof NoopSpan)) {
-      span.setTag("params", params.toString());
-    }
     SolrQueryRequest sreq = buildRequestFrom(core, params, streams, getRequestTimer(req), req);
 
     // Handlers and login will want to know the path. If it contains a ':'
     // the handler could use it for RESTful URLs
-    sreq.getContext().put(PATH, RequestHandlers.normalize(path));
+    String pathNormalized = RequestHandlers.normalize(path);
```

Comment:
why did you need to split the above row in half?

Reply:
The code placement of some of my changes moved in the course of multiple commits on this PR, but didn't leave things identical to how things were originally.  I will undo this.